### PR TITLE
include jars

### PR DESCRIPTION
**Pull request by ahorek**
```digest/digest.jar``` should be included

### CODE REVIEW

File: digest.gemspec [hunk 1]
```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   if Gem::Platform === spec.platform and spec.platform =~ 'java' or RUBY_ENGINE == 'jruby'
     spec.platform = 'java'
 
+    spec.files += Dir["lib/*.jar"]
     spec.files += Dir["ext/java/**/*.{rb,java}"]
```
Comment (olleolleolle) [inline digest.gemspec:30]: I think is  equivalent?

```suggestion
    spec.files += Dir["ext/java/**/*.{rb,java}", "lib/*.jar"]
```

**Comment by knu** [comment]
Thanks.  It's sad the integration test passed because the digest library would be still available even if the jar were missing. 🥺

File: digest.gemspec [hunk 2]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   if Gem::Platform === spec.platform and spec.platform =~ 'java' or RUBY_ENGINE == 'jruby'
     spec.platform = 'java'
 
-    spec.files += Dir["ext/java/**/*.{rb,java}"]
+    spec.files += Dir["ext/java/**/*.{rb,java}", "lib/*.jar"]
```
Comment (knu) [inline digest.gemspec:29]: Let's be specific here, like `"lib/digest.jar"`.

**Review by knu — CHANGES_REQUESTED** [review]
Please squash the commits into one.

**Comment by ahorek** [comment]
> It's sad the integration test passed because the digest library would be still available even if the jar were missing. 🥺

the library is available because it wasn't removed yet https://github.com/jruby/jruby/pull/6719/files

**Review by knu — APPROVED** [review]
Thank you!

**Pull request merged**